tests/main_test.py

```diff
@@ -53,6 +54,13 @@ def test_password_in_sha512():
     assert "password" in x
 
 
+def test_help_menu():
+    runner = CliRunner()
+    result = runner.invoke(main)
+    print(result.output)
+    assert "Search-That-Hash - The fastest way to crack any hash." in result.output
+
+
```

Comment:
This is literally the test that solves "Don't ignore this, you can 100% test whether or not the program has no input lol"

Reply:
Sure.Done :)